umlet-gwt/src/main/java/com/baselet/gwt/client/view/EventHandlingUtils.java

```diff
@@ -364,6 +365,10 @@ private static EventHandlingTarget getPanelWhichContainsPoint(EventHandlingTarge
 		EventHandlingTarget returnPanel = null;
 		for (EventHandlingTarget panel : panels) {
 			Rectangle visibleBounds = panel.getVisibleBounds();
+			// For some reason, MouseUpEvent does not always fire for same position as MouseDownEvent, therefore making
+			// check of point inside panel more narrow
+			visibleBounds.x += 1;
```

Comment:
Do you have an example where this was necessary? Was it just VSCode or also Umletino?
I'm just asking to avoid potential future errors where things may be displaced by some pixels from this change

Reply:
I could reproduce this issue in Firefox, Chromium and VSCode for the most recent versions of UMLet. If the diagram panel contained at least one UML element and the user then clicked on the leftmost part of the draggable layout-splitter dividing diagram and palette panel and released the click without moving the mouse afterwards, only onMouseDown fired initiating a drag of the diagram-element without stopping the drag due to the missing onMouseUp event. This caused the diagram element to be dragged by just moving the mouse around afterwards until another click (down+up) is initiated.